### PR TITLE
vest scores

### PR DESCRIPTION
**Pull request by bwalsh**
This PR has one new feature:
* adds vest scores
(Paired with https://github.com/biostream/vest-transform/pull/1 )

### CODE REVIEW

File: proto/bmeg/vest.proto [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+// vest score
+
+syntax = "proto3";
+
+package bmeg;
+
+message VestScore {
```
Comment (buchanae) [inline proto/bmeg/vest.proto:9]: Perhaps it's better to put this in a `vest` namespace (e.g. `package vest;`) and drop the `Vest` prefix here?

File: proto/bmeg/vest.proto [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+   string aa_mutation = 1;
+   string alt = 2;
+   string chromosome = 3;
+
+  message GENE {
```
Comment (buchanae) [inline proto/bmeg/vest.proto]: Why all caps?

File: proto/bmeg/vest.proto [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+     string id = 1;
+     string name = 2;
+  }
+
+   GENE gene = 4;
+   string protein_alt = 5;
+   string protein_ref = 6;
+   string ref = 7;
+   double score = 8;
+   sint64 start = 9;
```
Comment (buchanae) [inline proto/bmeg/vest.proto]: Unless there's a particular reason we need max efficiency here ([docs](https://developers.google.com/protocol-buffers/docs/encoding#signed-integers)), let's avoid inconsistency and use a regular `int64`

File: python/bmeg/rna_pb2.py [hunk 4]
```diff
@@ -84,7 +84,7 @@
     _descriptor.FieldDescriptor(
       name='value', full_name='bmeg.GeneExpression.ExpressionsEntry.value', index=1,
       number=2, type=1, cpp_type=5, label=1,
-      has_default_value=False, default_value=0,
+      has_default_value=False, default_value=float(0),
```
Comment (buchanae) [inline python/bmeg/rna_pb2.py:86]: Kinda strange all these changed. Any idea why?

Comment (bwalsh) [reply python/bmeg/rna_pb2.py:86]: no idea. 
is this the same version you use?
```
$ protoc --version
libprotoc 3.0.0
```

File: proto/bmeg/vest.proto (hunk 1 above)
Comment (bwalsh) [reply proto/bmeg/vest.proto:9]: Following the pattern, all the other files in the bmeg dir use `package bmeg`.  Don't know.  Perhaps `package bmeg.vest`?

File: python/bmeg/rna_pb2.py (hunk 4 above)
Comment (bwalsh) [reply python/bmeg/rna_pb2.py:86]: hmm .

updated to 
```
$ protoc --version
libprotoc 3.5.1
```

The ` default_value=float(0),` is still there, but now there are the following...



```
 _EXPRESSIONSCALE = _descriptor.EnumDescriptor(
   name='ExpressionScale',
@@ -80,14 +79,14 @@ _GENEEXPRESSION_EXPRESSIONSENTRY = _descriptor.Descriptor(
       has_default_value=False, default_value=_b("").decode('utf-8'),
       message_type=None, enum_type=None, containing_type=None,
       is_extension=False, extension_scope=None,
-      options=None),
+      options=None, file=DESCRIPTOR),
     _descriptor.FieldDescriptor(
       name='value', full_name='bmeg.GeneExpression.ExpressionsEntry.value', index=1,
       number=2, type=1, cpp_type=5, label=1,
       has_default_value=False, default_value=float(0),
       message_type=None, enum_type=None, containing_type=None,
       is_extension=False, extension_scope=None,
-      options=None),
+      options=None, file=DESCRIPTOR),
   ],


........

```

**Review by prismofeverything — APPROVED** [review]

**Comment by prismofeverything** [comment]
Seems to be conflicts, unable to merge from the dentist's office

**Pull request merged**